spec/controllers/show_projects_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+RSpec.describe ShowProjectsController, type: :controller do
+
+  describe "GET #index" do
```

Comment:
I think it would make sense to have a few more tests here. One could for instance test for the `:index` template to be rendered or, in my opinion even more meaningful for the two scopes being called on `Project`.

The latter would again be something like
```ruby
projects_of_season = double(accepted: [])
expect(Project)
  .to receive(:in_current_season)
  .and_return(projects_of_season)
expect(projects_of_season)
  .to receive(:accepted)
```

Note that the above may not be executable, I just wrote it as an inspiration out of memory.